theano/sandbox/cuda/__init__.py

```diff
@@ -304,6 +304,8 @@ def dnn_available():
             if config.nvcc.compiler_bindir:
                 params.extend(['--compiler-bindir',
                                config.nvcc.compiler_bindir])
+            [params.append(flag) for flag in config.nvcc.flags.split(' ') if flag] 
```

Comment:
It is working. But I think this version is more clear:

    params.extend([flag for flag in config.nvcc.flags.split(' ') if flag])

thanks.